### PR TITLE
tickets link

### PR DESCRIPTION
**Pull request by hunterowens**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,8 @@
 
   <p><a href="/sched.html"><strong>The list of talks has been announced!</strong></a> Check back soon for final talk times and dates..</p>
   <p>Not sure what to expect? Have a look at <a href="http://www.juliacon.org/2014">
-     last year's program and videos</a>.</p>
+      last year's program and videos</a>.</p>
+  <p><a href="http://www.eventbrite.com/e/juliacon-2015-tickets-16517619645">Tickets</a> are now available.</p>
```
Comment (IainNZ) [inline index.html]: Shouldn't be here, should be in the next `<div>` under `Registration`

**Comment by IainNZ** [comment]
:+1:

**Comment by andreasnoack** [comment]
What is holding this back?

**Comment by ViralBShah** [comment]
Bump. Can we merge this and announce?

**Comment by StefanKarpinski** [comment]
Yes, seems like a good idea.

**Comment by ViralBShah** [comment]
Thanks! Should we send out an announcement on registrations open to julia-users?

**Comment by andreasnoack** [comment]
I think that would be a good idea.

2015-05-08 13:36 GMT-04:00 Viral B. Shah <notifications@github.com>:

**Comment by ViralBShah** [comment]
I just posted to julia-users.

**Pull request merged**